src/Microsoft.Win32.Registry/tests/Microsoft.Win32.Registry.Tests.csproj

```diff
@@ -56,6 +59,11 @@
     <Compile Include="TestData.cs" />
     <Compile Include="XunitAssemblyAttributes.cs" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)'=='net46'">
```

Comment:
looks this is related to your first comment. how we can build the test as portable and then run it against the desktop.